libs/transfer/src/lib/views/make-transfer-view/make-transfer-view.component.html

```diff
@@ -1,4 +1,10 @@
 <ng-container *ngIf="(vm$ | async)! as data">
+  <ng-container *ngIf="errorAlert$ | async as error">
+    <ng-container
+      [ngTemplateOutlet]="errorTemplate"
```

Comment:
Good point!
I like to use `ngTemplateOutlet` because it's much cleaner rather than having the code block 
directly at the very beginning of the template.

I agree we could create a re-usable component if the code is big but in this case IMHO
`bb-alert-ui` is already re-usable and is not too big.

So I am ok with using it directly instead of having `ngTemplateOutlet`.
I have used this solution in one of the CS projects as well.

Let me know what do you think about it ;)

Reply:
Updated the PR with the above changes 

https://github.com/Backbase/golden-sample-app/pull/131/files/973f01bcff56719a67f71ffa0d106de007e07d56..1a0c64319a25b7a73c610723aa1a21ff1e0f7161